core/src/main/java/apoc/create/Create.java

```diff
@@ -220,6 +221,31 @@ public Relationship vRelationshipFunction(@Name("from") Node from, @Name("relTyp
         return Stream.of(new VirtualPathResult(from, rel, to));
     }
 
+    @Procedure
+    @Description("apoc.create.vPath")
```

Comment:
what's the difference with `apoc.create.virtualPath`?

Reply:
The `apoc.create.vPath` create a virtual Path from another real Path,
through a `PathImpl.Builder` that create everything I need.
With `apoc.create.virtualPath` we have to pass several parameters (`labels`, `relType`, `props`) to create our parameter,
so if I already have a path, I should model it to fit the `apoc.create.virtualPath`.

Reply:
what about renaming this procedure to something like `clonePathToVirtual`?